styles for section advantages and gallery in scss

Tate-T opened the pull request:
1. У галереї прийдеться обрізати усі картинки, тому що коли я кладу цілу картинку, то вона вжимається в контейнер і тоді краї дрібних картинок не співпадають з межею контейнера (як на макеті). А коли роблю позиціонування то всеодно ширина не підходить, бо я не буду розтягувати картинку більше, ніж за каонтейнер.
2. Не розумію чому у мене не відображаються png (як before), думаю що проблема з адресою, але не розумію як правильно вказати (хоча ми вказували так як я вказала).
3. у параграфі перебивається шрифт bold, передивилась стилі, не бачу що саме перебиває його.

Tate-T committed:
added styles to advantages

Tate-T committed:
added styles for gallery-section

yalanskyyalexey commented on src/sass/layout/_advantages.scss:
```diff
@@ -0,0 +1,88 @@
+    &__title {
+        margin-top: 0;
+        font-family: Titan One;
+        font-weight: 400px;
```
без px пожалуйста размеры

yalanskyyalexey commented on src/sass/layout/_advantages.scss:
```diff
@@ -0,0 +1,88 @@
+
+    &__title {
+        margin-top: 0;
+        font-family: Titan One;
```
для font-family используем переменную - $second-font

yalanskyyalexey commented on src/sass/layout/_advantages.scss:
```diff
@@ -0,0 +1,88 @@
+    }
+
+    &__text {
+        font-family: DM Sans;
```
переменная - $main-font

yalanskyyalexey commented on src/sass/layout/_advantages.scss:
```diff
@@ -0,0 +1,88 @@
+
+    &__text {
+        font-family: DM Sans;
+        font-weight: 700px;
```
без пикселей, проверить весь код на наличие подобных недочетов)

yalanskyyalexey reviewed (commented):
Исправить отмеченные замечания

Tate-T committed:
corrected mistakes

до речі, змінну для шрифту я вписувала, може глянув один з перших комітів;

а от з відображенням іконок треба розбиратись, завтра це і зроблю

Tate-T commented:
до речі, змінну для шрифту я вписувала, може глянув один з перших комітів;

а от з відображенням іконок треба розбиратись, завтра це і зроблю
